### PR TITLE
Fix console commands

### PR DESCRIPTION
**Pull request by iamsaint**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: src/Command/ListenCommand.php [hunk 1]
```diff
@@ -8,27 +8,36 @@
...
-        parent::__construct($name);
+        parent::__construct();
         $this->queueFactory = $queueFactory;
-        $this->setDescription('Listens the queue and executes messages as they come. Needs to be stopped manually.');
+    }
 
-        $this->addArgument('channel', InputArgument::OPTIONAL, 'Queue channel name to connect to', QueueFactory::DEFAULT_CHANNEL_NAME);
+    public function configure(): void
+    {
+        $this->setDescription('Listens the queue and executes messages as they come. Needs to be stopped manually.');
```
Comment (BoShurik) [inline src/Command/ListenCommand.php:28]: To make command lazy move description to `protected static $defaultDescription`

**Review by BoShurik — CHANGES_REQUESTED** [review]

File: src/Command/ListenCommand.php [hunk 2]
```diff
@@ -8,27 +8,36 @@
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
+use Yiisoft\Yii\Console\ExitCode;
```
Comment (vjik) [inline src/Command/ListenCommand.php:11]: Need add `yiisoft/yii-console` to requirements in `composer.json`

File: src/Command/ListenCommand.php (hunk 1 above)
Comment (iamsaint) [reply src/Command/ListenCommand.php]: done

Comment (BoShurik) [reply src/Command/ListenCommand.php]: `$this->setDescription(self::$defaultDescription)` is redundant

File: src/Command/ListenCommand.php [hunk 3]
```diff
@@ -8,27 +8,38 @@
...
-        parent::__construct($name);
+        parent::__construct();
         $this->queueFactory = $queueFactory;
-        $this->setDescription('Listens the queue and executes messages as they come. Needs to be stopped manually.');
+    }
 
-        $this->addArgument('channel', InputArgument::OPTIONAL, 'Queue channel name to connect to', QueueFactory::DEFAULT_CHANNEL_NAME);
+    public function configure(): void
+    {
+        $this->setDescription(self::$defaultDescription);
```
Comment (vjik) [inline src/Command/ListenCommand.php:30]: `setDescription() not need when use `static $defaultDescription`.

File: src/Command/RunCommand.php [hunk 4]
```diff
@@ -8,27 +8,38 @@
...
-        parent::__construct($name);
+        parent::__construct();
         $this->queueFactory = $queueFactory;
-        $this->setDescription('Runs all the existing messages in the queue. Exits once messages are over.');
+    }
 
-        $this->addArgument('channel', InputArgument::OPTIONAL, 'Queue channel name to connect to', QueueFactory::DEFAULT_CHANNEL_NAME);
+    public function configure(): void
+    {
+        $this->setDescription(self::$defaultDescription);
```
Comment (vjik) [inline src/Command/RunCommand.php:30]: `setDescription() not need when use `static $defaultDescription`.

**Review by vjik — CHANGES_REQUESTED** [review]

**Review by vjik — APPROVED** [review]

**Comment by vjik** [comment]
Conflict with `composer.json` from `master`. @iamsaint can you merge `master` branch to this PR?

**Review by samdark — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**